ol3-cesium-plugin/garastersynchronizer.js

```diff
@@ -34,38 +34,9 @@ ga.GaRasterSynchronizer.prototype.convertLayerToCesiumImagery =
   }
 
   // Read custom, non standard properties
-  var layerId = olLayer['id'];
-  var layerTime = olLayer['time'];
-
-  if (source instanceof ol.source.WMTS) {
-    provider = new Cesium.WebMapTileServiceImageryProvider({
-      url: 'https://mf-chsdi3.dev.bgdi.ch/mom_fix_1594/1.0.0/' +
-         layerId + '/{Style}/' + layerTime +
-         '/{TileMatrixSet}/{TileMatrix}/{TileCol}/{TileRow}.png',
-      layer : layerId,
-      style : 'default',
-      format : 'image/png',
-      tileMatrixSetID : '4326',
-      // Is a custom Cesium.GeographicTilingScheme necessary?
-      tilingScheme: new Cesium.GeographicTilingScheme({
-        numberOfLevelZeroTilesX: 2,
-        numberOfLevelZeroTilesY: 1,
-        rectangle: new Cesium.Rectangle(
-          Cesium.Math.toRadians(-180.0),
-          Cesium.Math.toRadians(-90.0),
-          Cesium.Math.toRadians(180.0),
-          Cesium.Math.toRadians(90.0))
-      }),
-      maximumLevel: 19
-    });
-  } else if (source instanceof ol.source.TileWMS ||
-             source instanceof ol.source.ImageWMS) {
-    provider = new Cesium.WebMapServiceImageryProvider({
-      url: source.getUrls ? source.getUrls()[0] :
-                            /** @type {string} */ (source.getUrl()),
-      layers: layerId,
-      parameters: {format:'image/png'}
-    });
+  var getProvider = olLayer['getCesiumImageryProvider'];
+  if (getProvider) {
```

Comment:
Technically an external WMS could be displayed in 3d  if the WMS support 1.1.1 so it should have one. And yes should not be thrown if `getProvider`is not defined.
I fix.